src/delta.rs

```diff
@@ -327,7 +327,6 @@ fn handle_generic_file_meta_header_line(
     raw_line: &str,
     config: &Config,
 ) -> std::io::Result<()> {
-    // FIXME: this may be able to be refactored by moving to should_handle.
```

Comment:
Sorry, I added fixme, but I realized this can't be refactored.

When file style is assinged to `omit` we should return OK, however, if it's while color_only mode, we shouldn't return OK.

If we move this condition to should_handle it would be messy because not only State::FileMeta is using should_handle.